src/subcommands.rs

```diff
@@ -0,0 +1,5 @@
+/// TODO: Add host and vm subcommands here?
```

Comment:
This is not needed. Make a directory called subcommands, move host.rs and vm.rs (renamed to api.rs, see above) into that directory, then add public module declarations for them here. Besides that this file should be empty I think.